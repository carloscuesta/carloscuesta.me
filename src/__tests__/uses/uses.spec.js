import renderer from 'react-test-renderer'

import Uses from 'src/pages/uses'

jest.mock('next-seo', () => ({ NextSeo: 'NextSeo' }))

/*
  I'm mocking the components since those are unit tested.
  So there's no point to test the rendering of them twice.
  It's more easier to manage little snapshots than big ones.
*/
jest.mock('src/components/pages/uses/Section', () => 'Section')

describe('uses', () => {
  describe('page', () => {
    it('should match uses page', () => {
      const wrapper = renderer.create(<Uses />)

      expect(wrapper).toMatchSnapshot()
    })
  })
})
