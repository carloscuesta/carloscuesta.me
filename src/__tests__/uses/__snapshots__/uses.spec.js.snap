// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uses page should match uses page 1`] = `
Array [
  <NextSeo
    canonical="https://carloscuesta.me/uses"
    title="Carlos Cuesta â€“ Uses"
  />,
  <main>
    <div
      className="wrapper"
    >
      <h1
        className="title"
      >
        Uses
      </h1>
      <div
        className="row"
      >
        <div
          className="col-xs-12"
        >
          <SetupImage />
        </div>
        <div
          className="col-xs-12"
        >
          <Section
            items={
              Array [
                Object {
                  "category": "Desk",
                  "link": "https://www.fully.com/standing-desks/jarvis-adjustable-height-desk-bamboo.html",
                  "title": "Fully Jarvis Bamboo Standing Desk",
                },
                Object {
                  "category": "Laptop",
                  "link": "https://support.apple.com/kb/sp704",
                  "title": "MacBook Pro 15 (Mid 2014), i7 @ 2,5GHz, 16 GB",
                },
                Object {
                  "category": "Chair",
                  "link": "https://www.ikea.com/es/es/p/markus-silla-trabajo-glose-negro-40103100/",
                  "title": "Ikea Markus",
                },
                Object {
                  "category": "Monitor",
                  "link": "https://www.dell.com/es-es/shop/monitor-curvo-dell-ultrasharp-38-con-concentrador-y-usb-c-u3821dw/apd/210-axnt/monitores-y-accesorios",
                  "title": "Dell U3821DW",
                },
                Object {
                  "category": "Webcam",
                  "link": "https://bit.ly/3jDlMZp",
                  "title": "Logitech C920",
                },
                Object {
                  "category": "Keyboard",
                  "link": "https://bit.ly/3AonlRy",
                  "title": "Keychron K3 V2",
                },
                Object {
                  "category": "Mouse",
                  "link": "https://bit.ly/3iAMqTr",
                  "title": "Logitech Triathlon M720",
                },
                Object {
                  "category": "Desk Matt",
                  "link": "https://bit.ly/3CvZy40",
                  "title": "Eco Leather Desk Matt",
                },
                Object {
                  "category": "Headphones",
                  "link": "https://www.apple.com/airpods-pro",
                  "title": "Airpods Pro",
                },
                Object {
                  "category": "Lighting",
                  "link": "https://www.benq.com/en-us/lamps/computer-desklamp/screenbar.html",
                  "title": "BenQ Screenbar",
                },
                Object {
                  "category": "Headphones",
                  "link": "https://www.bose.com/en_us/products/headphones/noise_cancelling_headphones/quietcomfort-headphones-45.html#v=qc45_white_smoke",
                  "title": "Bose QuietComfort 45",
                },
              ]
            }
            subTitle="Home office equipment"
            title="Gear"
          />
          <Section
            items={
              Array [
                Object {
                  "category": "Text Editor",
                  "link": "https://atom.io",
                  "title": "Atom",
                },
                Object {
                  "category": "Terminal",
                  "link": "https://iterm2.com",
                  "title": "iTerm2",
                },
                Object {
                  "category": "Dotfiles",
                  "link": "https://github.com/carloscuesta/dotfiles",
                  "title": "Carlos Cuesta Dotfiles",
                },
                Object {
                  "category": "Window Management",
                  "link": "https://rectangleapp.com",
                  "title": "Rectangle",
                },
                Object {
                  "category": "JS Playground",
                  "link": "http://runjs.app",
                  "title": "RunJS",
                },
                Object {
                  "category": "Design",
                  "link": "http://figma.com",
                  "title": "Figma",
                },
              ]
            }
            subTitle="Tools to work efficiently"
            title="Software"
          />
          <Section
            items={
              Array [
                Object {
                  "category": "Microphone",
                  "title": "Blue Yeti X / Elgato Wave",
                },
                Object {
                  "category": "Anti-fatigue standing mat",
                  "link": "https://en.aeris.de/products/aeris-muvmat-schwarz-ohne-bezug",
                  "title": "Muvmat",
                },
                Object {
                  "category": "Monitor Arm",
                  "link": "https://www.ergotron.com/es-es/productos/detalles-del-producto/45-475",
                  "title": "Ergotron HX",
                },
              ]
            }
            subTitle="Things to improve the setup"
            title="Wishlist"
          />
        </div>
      </div>
    </div>
  </main>,
]
`;
